src/base/tests/test_util.py

```diff
@@ -28,6 +28,9 @@
 from odoo.addons.base.maintenance.migrations.util.exceptions import MigrationError
 
 
+USE_ORM_DOMAIN = util.misc.version_gte("saas~18.2")
+
```

Comment:
Most of the diff is about the `"!", "!"` that are removed. We can handle these with unpacking.
```suggestion
USE_ORM_DOMAIN = util.misc.version_gte("saas~18.2")
NOTNOT = () if USE_ORM_DOMAIN else ("!", "!")
```